gremlin-language/src/main/antlr4/Gremlin.g4

```diff
@@ -802,6 +803,11 @@ traversalMethod_call
     | 'call' LPAREN stringArgument COMMA genericLiteralMapArgument COMMA nestedTraversal RPAREN #traversalMethod_call_string_map_traversal
     ;
 
+traversalMethod_concat
+	: 'concat' LPAREN RPAREN #traversalMethod_concat_Empty //is this needed given stringLiteralList?
```

Comment:
Appears to be redundant considering
```
| 'concat' LPAREN stringLiteralVarargs RPAREN #traversalMethod_concat_String
```
and
```
stringLiteralVarargs
    : (stringNullableArgument (COMMA stringNullableArgument)*)?
```

I would suggest removing this as long as we test parsing `concat()` via the grammar at some point.